### PR TITLE
Fix charset in inline html file

### PR DESCRIPTION
**Pull request by DimaAmega**

### CODE REVIEW

File: src/evidently/suite/base_suite.py [hunk 1]
```diff
@@ -253,9 +261,9 @@ def save_html(self, filename: Union[str, IO], mode: Union[str, SaveMode] = SaveM
                 dashboard_info=dashboard_info,
                 additional_graphs=graphs,
             )
-            render = self._render(determine_template("inline"), template_params)
+            render = self._render(file_html_template, template_params)
             if isinstance(filename, str):
-                with open(filename, "w", encoding="utf-8-sig") as out_file:
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:258]: No longer need `utf-8-sig` charset

File: src/evidently/suite/base_suite.py [hunk 2]
```diff
@@ -274,8 +282,8 @@ def save_html(self, filename: Union[str, IO], mode: Union[str, SaveMode] = SaveM
                 font_file=font_file,
                 include_js_files=[lib_file, data_file],
             )
-            with open(filename, "w", encoding="utf-8-sig") as out_file:
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:277]: No longer need `utf-8-sig` charset

File: src/evidently/utils/dashboard.py [hunk 3]
```diff
@@ -140,57 +101,67 @@ def file_html_template(params: TemplateParams):
...
-window.drawDashboard({params.dashboard_id},
-    new Map(Object.entries(additional_graphs_{params.dashboard_id})),
-    "root_{params.dashboard_id}"
-);
-</script>
-</body>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
```
Comment (DimaAmega) [inline src/evidently/utils/dashboard.py:144]: `utf-8` charset here

File: src/evidently/suite/base_suite.py [hunk 4]
```diff
@@ -253,9 +261,9 @@ def save_html(self, filename: Union[str, IO], mode: Union[str, SaveMode] = SaveM
                 dashboard_info=dashboard_info,
                 additional_graphs=graphs,
             )
-            render = self._render(determine_template("inline"), template_params)
+            render = self._render(file_html_template, template_params)
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:264]: Pay attention that we are using `file_html_template` now here with `<meta charset="UTF-8">` tag inside

File: src/evidently/suite/base_suite.py [hunk 5]
```diff
@@ -274,8 +282,8 @@ def save_html(self, filename: Union[str, IO], mode: Union[str, SaveMode] = SaveM
                 font_file=font_file,
                 include_js_files=[lib_file, data_file],
             )
-            with open(filename, "w", encoding="utf-8-sig") as out_file:
-                out_file.write(self._render(determine_template("inline"), template_params))
+            with open(filename, "w", encoding="utf-8") as out_file:
+                out_file.write(self._render(file_html_template, template_params))
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:286]: same as [here](https://github.com/GwenVCX/evidently/pull/1/files#r1677878050)

File: src/evidently/suite/base_suite.py [hunk 6]
```diff
@@ -227,7 +231,7 @@ def show(self, mode="auto"):
         try:
             from IPython.display import HTML
 
-            return HTML(self._render(determine_template(mode), template_params))
+            return HTML(self._render(inline_iframe_html_template, template_params))
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:234]: we are using `iframe` (not like just embed all js and html stuff as it is) in order to render UI in jupyter notebook

File: src/evidently/suite/base_suite.py [hunk 7]
```diff
@@ -207,7 +211,7 @@ def _repr_html_(self):
             dashboard_info=dashboard_info,
             additional_graphs=graphs,
         )
-        return self._render(determine_template("auto"), template_params)
+        return self._render(inline_iframe_html_template, template_params)
```
Comment (DimaAmega) [inline src/evidently/suite/base_suite.py:214]: same as [here](https://github.com/GwenVCX/evidently/pull/1/files#r1677883334)

**Review by GwenVCX — APPROVED** [review]
Hi @DimaAmega,

Thank you for those updates. I am back from annual leave so I will merge this PR into my evidently fork.

Cheers,

Gwen

**Pull request merged**